TROUBLESHOOTING.md

```diff
@@ -0,0 +1,70 @@
+Troubleshooting the plugin
+==============
+
+### Verifying the setup
+
+**0.** ensure the latest compatible version of IntelliJ IDEA Community Edition is installed (**2020.2.** or later).
+
+**1.** update plugin to the latest version (you can check the latest version on [Plugin's official page](https://plugins.jetbrains.com/plugin/13634-a-courses/versions)), but at least **1.4**.
+
+**2.** create a new project: [**File -> New -> Project...** and then **Empty Project**](https://plus.cs.aalto.fi/o1/2020/w01/ch02/#launch-intellij).
+
+**3.** Turn project into an A+ Project: [**A+ -> Turn Project Into A+ Project**](https://plus.cs.aalto.fi/o1/2020/w01/ch02/#the-a-courses-plugin) (and choose the rest of the settings asked for in the pop-up dialog).
+
+**4.** check the IDE for not having errors (you **should not** see the blinking red icon as shown). If there are errors, please share the "stacktrace" text with the course staff.
+![libraries](images/error_.png)
+
+**5.** review the project structure for correctness: **File -> Project Structure ->** ...
+
+**Project**
+
+![project](images/ts_project.png)
+
+**Modules**
+![modules 3](images/ts_modules_3.png)
+![modules 1](images/ts_modules_1.png)
+![modules 2](images/ts_modules_2.png)
+
+**Libraries (Scala SDK)**
+![libraries](images/ts_libs.png)
+
+**6.** download a **GoodStuff** module and run it by right-clicking and choosing **Run**
+![run](images/Run.png)
+
+**7.** start REPL on a **GoodStuff** module by right-clicking **Scala REPL...**
+![repl](images/REPL.png)
+
+### [Known issues](https://github.com/Aalto-LeTech/intellij-plugin/labels/user-bug)
+
+| #   | status    | description                                                                                        |
+|-----|-----------|----------------------------------------------------------------------------------------------------|
+| 1.  | **problem**   | [~~IDE complaining about not able to see the SDK (whitespaces and special characters in naming (path))~~](https://github.com/Aalto-LeTech/intellij-plugin/issues/360)|
```

Comment:
Isn't the actual error something like 
> scalac: Scala compiler JARs not found (module: '<module name>')

Also, since we have "status", let's not use strikethrough here since it's a bit difficult to read